cmd/down.go

```diff
@@ -89,6 +89,7 @@ func runDown(ctx context.Context, dev *model.Dev) error {
 	spinner := utils.NewSpinner("Deactivating your development container...")
 	spinner.Start()
 	defer spinner.Stop()
+	go utils.StopSpinnerIfInterruptSignal(spinner)
```

Comment:
could we execute this as part of `spinner.Stop()`?

Reply:
If we do that we could not run the shutdown sequence on `okteto up` command, because we need to return signal exit code, otherwise what `CTRL+C` will do is just running `spinner.Stop()`

Reply:
Can we have two `spinner.Stop` functions and call one or the other depending on the command?